flash.py

```diff
@@ -19,7 +19,7 @@ def fmtcols(mylist, cols):
 
 def progress_report(devices):
     """ Print a list of devices and their flashing status """
-    formatted_status = fmtcols(["[{status}] {d}".format(status = 'X' if dev.flashed == True else ' ', d=dev.name) for dev in devices], 3)
+    formatted_status = fmtcols(["[{status}] {d}".format(status = 'X' if (dev.flashed == True and dev.online == True) else ' ', d=dev.name) for dev in devices], 3)
```

Comment:
Do we want a different thing if it was flashed, but hasn't come online?

Reply:
Maybe, but in both cases I'd probably just reflash it. I think reworking the progress reports and other printouts will be a separate thing I rework though.

Reply:
Makes sense.